pkg/odo/cli/application/application.go

```diff
@@ -23,6 +24,12 @@ var (
 	applicationForceDeleteFlag bool
 )
 
+// Desc holds all information about application
+type AppDesc struct {
```

Comment:
Can we please change this as well?

Also I would prefer to use Description instead of Desc both here and for the component. WDYT @metacosm ?

Reply:
done